Dockerfile

```diff
@@ -19,38 +19,37 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
         libpng-dev \
         python3 \
         python3-pip \
-        python3-venv \
         python3-dev && \
     rm -rf /var/lib/apt/lists/*
+# Remove PEP 668 restriction (safe in containers)
```

Comment:
This is required in order to install everything into default env

Reply:
But you also want to make the same change to the `official` section